### PR TITLE
Added check for RabbitMQ

### PR DESCRIPTION
**Pull request by darbyfrey**
Added a quick check that tests for a successful connection to RabbitMQ. This check requires the Bunny gem as well.

### CODE REVIEW

**Comment by pbyrne** [comment]
Would love to see a test around this. :grin: 

Such a test would also highlight that the file needs to be `require`d in `lib/okcomputer.rb` in the increasingly disturbing list of `require`s (which will be ultimately phased out with https://github.com/sportngin/okcomputer/issues/56#issuecomment-187219684, but that's another conversation for another day).

**Comment by darbyfrey** [comment]
Added specs  :tada: :balloon:

File: lib/ok_computer/built_in_checks/rabbitmq_check.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    def check
+      mark_message "Connected Successfully"
+      mark_message "Rabbit Connection Status: (#{connection_status})"
+    rescue => e
+      mark_failure
+      mark_message "Error: '#{e}'"
+    end
+
+    def connection_status
+      connection = Bunny.new(ENV['CLOUDAMQP_URL'] || ENV['AMQP_HOST'])
```
Comment (pbyrne) [inline lib/ok_computer/built_in_checks/rabbitmq_check.rb]: All things being equal, I think I'd prefer this URL to be passed in the initializer vs inferred within the check, _a la_ [the Redis check](https://github.com/sportngin/okcomputer/blob/22989b00b9a717d7f102414d6667dee2cc4c62aa/lib/ok_computer/built_in_checks/redis_check.rb#L15-L17).

Comment (darbyfrey) [inline lib/ok_computer/built_in_checks/rabbitmq_check.rb]: Easy enough to fix.

What do you think about this change which will allow for both?

Comment (pbyrne) [inline lib/ok_computer/built_in_checks/rabbitmq_check.rb]: Even better than just an initializer. Allow the user to specify, but fall back to a reasonable default. I'm unfamiliar with RabbitMQ, but are those common environment variables used to configure it?

Comment (darbyfrey) [inline lib/ok_computer/built_in_checks/rabbitmq_check.rb]: Yea, they are fairly common. There isn't a ton of consistency in that world yet, but these would cover us the most common cases. 

CloudAMQP is the big Heroku addon and they set `CLOUDAMQP_URL` when provisioning. https://devcenter.heroku.com/articles/cloudamqp#use-with-ruby

**Comment by pbyrne** [comment]
Merged and released as v1.7.0.

**Pull request merged**